file-formats/saia/type/zif_aff_saia_v1.intf.abap

```diff
@@ -0,0 +1,142 @@
+INTERFACE zif_aff_saia_v1
+  PUBLIC.
+
+  TYPES:
+    "! <p class="shorttext">Resource Type</p>
+    "! Resource type
+    BEGIN OF ty_resource_type,
+      "! <p class="shorttext">Category Scheme</p>
+      "! Category scheme
+      category_scheme TYPE string,
+      "! <p class="shorttext">Category Term</p>
+      "! Category term
+      category_term   TYPE string,
+    END OF ty_resource_type.
+
+  "! <p class="shorttext">Object Type in Object Directory</p>
+  "! Object Type in Object Directory
+  TYPES ty_trobjtype  TYPE c LENGTH 4.
+  "! <p class="shorttext">WB Request: (Internal) Type of an ABAP Workbench Object</p>
+  "! WB Request: (Internal) Type of an ABAP Workbench Object
+  TYPES ty_seu_objtyp TYPE c LENGTH 3.
+
+  TYPES:
+    "! <p class="shorttext">Global Workbench Object Type</p>
+    "! Global Workbench Object Type
+    BEGIN OF ty_wbobjtype,
+      "! <p class="shorttext">Object Type in Object Directory</p>
+      "! Object Type in Object Directory
+      objtype_tr TYPE ty_trobjtype,
```

Comment:
We recommend not to use abbreviations. Furthermore, the title (Shorttext) and the name of the component are ideally in sync